### PR TITLE
PLANET-7300: Move Accordion block into master theme

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: [PLANET-7300](https://jira.greenpeace.org/browse/PLANET-7300)

[Demo page](https://www-dev.greenpeace.org/test-phobos/accordion-block-demo-page/)

Move the Accordion block to master-theme

Related to [#1172](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1172)

### CODE REVIEW

File: assets/src/blocks/Accordion/AccordionEditor.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    } else {
+      newTabs[index][attributeName] = value;
+    }
+    setAttributes({
+      tabs: newTabs,
+    });
+  };
+
+  return (
+    <Fragment>
```
Comment (Osong-Michael) [inline assets/src/blocks/Accordion/AccordionEditor.js:185]: we can use the `<> </>` without the `Fragment`.

File: assets/src/scss/blocks/Accordion/AccordionEditorStyle.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  cursor: pointer;
+  font-weight: 500;
+  color: #0075af;
+  width: fit-content;
+  margin-top: 24px;
+
+  .plus {
+    border: 1px solid #0075af;
+    color: #0075af;
+    padding: 3px 8px;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionEditorStyle.scss:11]: we can use spacing variables here

File: assets/src/scss/blocks/Accordion/AccordionEditorStyle.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
+.add-button {
+  cursor: pointer;
+  font-weight: 500;
+  color: #0075af;
+  width: fit-content;
+  margin-top: 24px;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionEditorStyle.scss:6]: same here for margins

File: assets/src/scss/blocks/Accordion/AccordionStyle.scss [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+        html[dir="rtl"] & {
+          right: auto;
+          left: 16px;
+        }
+      }
+    }
+
+    .panel {
+      padding: 0 16px 24px 16px;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionStyle.scss:88]: same here, can use spacing variables

File: assets/src/scss/blocks/Accordion/AccordionStyle.scss [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+      line-height: 2.5;
+      width: 80%;
+    }
+  }
+
+  @include medium-and-up {
+    .accordion-content {
+      .accordion-headline {
+        padding-inline-start: 24px;
+        font-size: $font-size-md;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionStyle.scss:118]: we can use the appropriate token here

File: assets/src/scss/blocks/Accordion/AccordionStyle.scss [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+          right: 24px;
+
+          html[dir="rtl"] & {
+            left: 24px;
+          }
+        }
+      }
+
+      .panel {
+        padding: 0 24px 40px 24px;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionStyle.scss:133]: same here

File: assets/src/scss/blocks/Accordion/AccordionStyle.scss [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+        &:hover {
+          background: var(--p4-dark-green-900);
+        }
+      }
+      cursor: pointer;
+      position: relative;
+      display: block;
+      padding-top: 16px;
+      padding-bottom: 16px;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionStyle.scss:46]: You can use spacing variables here too

File: assets/src/scss/blocks/Accordion/AccordionStyle.scss [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+      padding-inline-start: 16px;
+      padding-inline-end: 44px;
+      font-size: $font-size-sm;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionStyle.scss:49]: We don't have this variable in the tokens file, can we replace it with the appropriate token?

File: assets/src/scss/blocks/Accordion/AccordionStyle.scss [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+      border-radius: 4px;
+      font-weight: 500;
+      line-height: 1;
+      margin-top: 0;
+      margin-bottom: 16px;
```
Comment (Osong-Michael) [inline assets/src/scss/blocks/Accordion/AccordionStyle.scss:54]: can use spacing values here too

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Pull request merged**